aviatrix/resource_aviatrix_edge_neo_device_onboarding.go

```diff
@@ -123,11 +123,11 @@ func marshalEdgeNEODeviceOnboardingInput(d *schema.ResourceData) *goaviatrix.Edg
 		network1 := network0.(map[string]interface{})
 
 		network2 := &goaviatrix.EdgeNEODeviceNetwork{
-			InterfaceName: network1["interface_name"].(string),
-			EnableDhcp:    network1["enable_dhcp"].(bool),
-			GatewayIp:     network1["gateway_ip"].(string),
-			Ipv4Cidr:      network1["ipv4_cidr"].(string),
-			ProxyServerIp: network1["proxy_server_ip"].(string),
+			InterfaceName:  network1["interface_name"].(string),
+			EnableDhcp:     network1["enable_dhcp"].(bool),
```

Comment:
We still need to update the markdown docs for this resource. Please take a look at the Docs folder.

Reply:
Sure. Will update the docs in the next PR

Reply:
Will fix the docs in the next PR